src/code.cloudfoundry.org/vendor/code.cloudfoundry.org/policy_client/internal_policy_client.go

```diff
@@ -56,6 +56,15 @@ func (c *InternalClient) GetPolicies() ([]*Policy, error) {
 	return policies.Policies, nil
 }
 
+func (c *InternalClient) GetPoliciesLastUpdated() (int, error) {
```

Comment:
Should we be adding this vendored code in the PR itself or just push the CI button to bump go mod deps after merging the PR?

This definitely helped for testing but idk what will happen after merging these changes.

Reply:
I think we can vendor manually this one and update the commit after policy client is merged, to have a working commit

Reply:
Oh, I just realized I committed vendor

Reply:
I rebased this commit with develop and it resolved our problem. Resolving this conversation.